### PR TITLE
Merging maizetrack into phenomenal

### PR DESCRIPTION
**Pull request by AurelienBesnier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,9 +57,9 @@ Conda environement : https://docs.conda.io/en/latest/index.html
 #### From source
 
     # Install dependency with conda
-    conda create -n phm -c conda-forge python=3.8
+    conda create -n phm -c conda-forge python
     conda activate phm
-    conda install -c conda-forge -c openalea cython numpy numba scipy scikit-image scikit-learn networkx=2.3 opencv matplotlib vtk pytest
+    conda install -c conda-forge -c numba cython numpy numba scipy scikit-image scikit-learn networkx opencv matplotlib vtk pytest skan=0.10
```
Comment (pradal) [inline README.md:62]: use mamba rather than conda

File: conda/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package:
   name: openalea.phenomenal
-  version: "1.7.1"
+  version: "1.9.2"
```
Comment (pradal) [inline conda/meta.yaml:3]: OK for the moment.
But a better solution is to get the version used in the setup.
See other openalea packages (eg. openalea.mtg)

File: setup.py [hunk 3]
```diff
@@ -18,12 +18,12 @@
...
 import numpy
 from Cython.Build import cythonize
-from setuptools import setup, find_packages, Extension, Command
+from setuptools import setup, find_namespace_packages, Extension
 # ==============================================================================
 
 namespace = "openalea"
 pkg_root_dir = 'src'
-packages = [pkg for pkg in find_packages(pkg_root_dir)]
+packages = find_namespace_packages(where='src', include=['openalea', 'openalea.*'])
```
Comment (pradal) [inline setup.py:26]: remove openalea from the namespace:
```python
find_namespace_packages(where='src', include=['openalea.*'])
```

File: setup.py [hunk 4]
```diff
@@ -18,12 +18,12 @@
...
 top_pkgs = [pkg for pkg in packages if len(pkg.split('.')) <= 2]
```
Comment (pradal) [inline setup.py:27]: remove all these lines 
```python
top_pkgs
```
and replace package_dir with
```python
package_dir={'': 'src'}
```

**Review by pradal — CHANGES_REQUESTED** [review]
Just consider my comments (setup.py and meta.yaml)

**Comment by AurelienBesnier** [comment]
@pradal I made the changes you asked. However, for the meta.yml, I left it the way it is for now as it failed for some reason (no module named 'numpy').

**Review by pradal — APPROVED** [review]

**Pull request merged**